src/main/java/mods/railcraft/common/util/misc/MiscTools.java

```diff
@@ -83,6 +86,23 @@ public static UUID readUUID(NBTTagCompound data, String tag) {
         return null;
     }
 
+    public static void writeUUID(DataOutputStream data, UUID uuid) throws IOException {
```

Comment:
These should probably be in DataTools in the network package rather than MiscTools.